### DISCUSSION TITLE
APISIX dashboard

### DISCUSSION THREAD

**Discussion by hariprasanth3031** [apache/apisix#9344] [Q&A]
How can I manage the apis created? Like view all the created apis

**Comment by ebrahimghane** [comment]
I think if you mean API design tools, Apisix is not the right choice. You can use API Curio Design, which has a publish with Redoc functionality. If you want exploratory and contributory functionality on APIs, you need an API developer portal tool. Apisix is only a tool to define APIs (routes, methods, upstream servers, etc.) in a web UI dashboard and configure an Nginx server to act as an API gateway.
remember that Apisix doesn't support SSO login for its dashboard currently
